libraries/Microsoft.Bot.Builder/IPropertyContainer.cs

```diff
@@ -1,39 +0,0 @@
-﻿using System;
```

Comment:
This went in the opposite direction than I imagined... 🤔 

BotState is now even more heavily violating the Single Responsibility Principle because it's no longer just that you chose to interface there, literally the APIs now belong to that type.